api/sql/schema.js

```diff
@@ -3,10 +3,11 @@ const path = require('path');
 module.exports = function sql({config}) {
     return {
         namespace: 'db/rule',
-        schema: [
-            {path: path.join(__dirname, 'schema'), linkSP: true, config},
-            {path: path.join(__dirname, 'schema/seeds')}
```

Comment:
schema/seeds folder is deleted, because it contained only one file, which duplicates what `750-user.action.merge.yaml` is doing